app/src/auth/utils/loginInterceptor.js

```diff
@@ -99,27 +99,27 @@ export function canHandleLoadError(err) {
     return false
 }
 
-export async function handleLoadError(err) {
-    if (err instanceof InvalidHexStringError) {
-        throw new ResourceNotFoundError(ResourceType.PRODUCT, err.id)
+export async function handleLoadError({ error, ignoreUnauthorized = false } = {}) {
+    if (error instanceof InvalidHexStringError) {
+        throw new ResourceNotFoundError(ResourceType.PRODUCT, error.id)
     }
 
-    const { status } = err.response || {}
+    const { status } = error.response || {}
 
     if (!status || status >= 500) {
-        throw err
+        throw error
     }
 
-    if (status === 404 || ([401, 403].includes(status) && isLoggedInError(err))) {
-        const data = err.response.data || {}
-        throw new ResourceNotFoundError(data.type, data.id)
+    if (status === 404 || ([401, 403].includes(status) && (!!ignoreUnauthorized || isLoggedInError(error)))) {
+        const data = error.response.data || {}
+        throw new ResourceNotFoundError(data.type || data.resource, data.id)
```

Comment:
Resource type is inconsistent in the backend, can be `type` or `resource`.

For example `/streams/id`:

<img width="712" alt="Screen Shot 2020-06-15 at 15 39 32" src="https://user-images.githubusercontent.com/1064982/84660900-3455ab80-af22-11ea-82eb-969d70dd9957.png">

`/streams/id/permissions/me`:

<img width="672" alt="Screen Shot 2020-06-15 at 15 38 17" src="https://user-images.githubusercontent.com/1064982/84660918-3a4b8c80-af22-11ea-924a-983165fde3fd.png">